hpx/runtime/parcelset/parcelport_impl.hpp

```diff
@@ -795,6 +846,20 @@ namespace hpx { namespace parcelset
             }
         }
 
+        void immediate_send_done(
+            boost::system::error_code const& ec,
+            locality const& locality_id,
+            std::shared_ptr<connection> sender_connection)
+        {
+            HPX_ASSERT(operations_in_flight_ != 0);
+            --operations_in_flight_;
+
+#if defined(HPX_TRACK_STATE_OF_OUTGOING_TCP_CONNECTION)
+            client_connection->set_state(parcelport_connection::state_scheduled_thread);
```

Comment:
It probable should be - but if that is the case, then the place I copied it from 
https://github.com/STEllAR-GROUP/hpx/blob/master/hpx/runtime/parcelset/parcelport_impl.hpp#L807
should also be changed - yes?